### DISCUSSION TITLE
Operationplan's attribute

### DISCUSSION THREAD

**Discussion by matteocarcano** [frePPLe/frepple#617] [Q&A]
Hi,
I would like to create operationplan's attribuite but when i do i can edit the new field but if i save changes i lost what i've just written, in other words frepple doesn't save changes. If i do the same thing with item's attribute it works correctly.

**Comment by jdetaeye** [comment]
Looks like a bug.

**Reply by matteocarcano** [reply]
Is it possible will you fix it?

**Reply by jdetaeye** [reply]
It'll be fixed in the next release.

**Reply by matteocarcano** [reply]
Thank you

**Reply by matteocarcano** [reply]
Hi,
i've tested after 8.30 release but bug is not fixed.

I've the same problem: Frepple doesn't save custom attribute changes.
